chaintesting/action_test_helpers.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package chaintesting
+
+import (
+	"context"
+	"testing"
+
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/codec"
+	"github.com/ava-labs/hypersdk/state"
+)
+
+var _ state.Mutable = (*InMemoryStore)(nil)
+
+// InMemoryStore is a storage that acts as a wrapper around a map and implements state.Mutable.
+type InMemoryStore struct {
+	Storage map[string][]byte
+}
```

Comment:
Could we put `InMemoryStore` in the same package as `state.Mutable` ?

Could we also export the constructor, so that tests can create a new in-memory store, but leave the type un-exported?

Reply:
I don't know if we want it to be in a production package, or maybe moved to the `state` package with some compilation tags ?